index.html

```diff
@@ -0,0 +1,2199 @@
+                        </a>
+                      </div>
+                    </div>
+                    <div class="d-flex align-items-center py-2 ">
+                      <span class="cover-span p-1">COVER</span>
+                      <a href="#" class="mag_link font-weight-bold px-2">NEWS</a>
+                    </div>
+                    <div>
+                      <a href="#">
+                        <h2 class="mag_link2">
+                          Who Should Doctors Save? Inside the Debate Over Rationing Coronavirus Care
+                        </h2>
+                      </a>
+                    </div>
+                  </article>
+                  <article class="col-sm-12 col-md-6 col-lg-3 w-100 " >
+                    <div class="w-100">
+                      <div class="w-100 ">
+                        <a href="#">
+                          <img class="w-100  full-height" src="https://d.newsweek.com/en/full/1579893/btn-sports-banner-basketball.jpg" alt="">
+                        </a>
+                      </div>
+                    </div>
+                    <div class="d-flex align-items-center py-2 ">
+                      <span class="cover-span p-1">COVER</span>
+                      <a href="#" class="mag_link font-weight-bold px-2">NEWS</a>
+                    </div>
+                    <div>
+                      <a href="#">
+                        <h2 class="mag_link2">
+                          Who Should Doctors Save? Inside the Debate Over Rationing Coronavirus Care
+                        </h2>
+                      </a>
+                    </div>
+                  </article>
+                  <article class="col-sm-12 col-md-6 col-lg-3 w-100 " >
+                    <div class="w-100">
+                      <div class="w-100 ">
+                        <a href="#">
+                          <img class="w-100  full-height" src="https://d.newsweek.com/en/full/1579893/btn-sports-banner-basketball.jpg" alt="">
+                        </a>
+                      </div>
+                    </div>
+                    <div class="d-flex align-items-center py-2 ">
+                      <span class="cover-span p-1">COVER</span>
+                      <a href="#" class="mag_link font-weight-bold px-2">NEWS</a>
+                    </div>
+                    <div>
+                      <a href="#">
+                        <h2 class="mag_link2">
+                          Who Should Doctors Save? Inside the Debate Over Rationing Coronavirus Care
+                        </h2>
+                      </a>
+                    </div>
+                  </article>
+                </div>
+            </section>
+          </div>
+      </div>
+
+      <div class="row">
+        <div class=" row d-flex align-items-center justify-content-start font-roboto signup-call py-3">
+          <h2 class="us px-2 mag_link font-weight-bold">U.S.</h2>
+        </div>
+        <div class="row">
```

Comment:
This section's three side articles have a different size (see how the pass the two articles on the left). Please make them look like the original page.
Also, include the rest of the sections that are missing.
Yours:
![Screenshot from 2020-04-22 13-21-10](https://user-images.githubusercontent.com/51926937/80019038-8804c500-849c-11ea-99e2-00501cfe4d69.png)

Original:
![Screenshot from 2020-04-22 13-21-28](https://user-images.githubusercontent.com/51926937/80019046-8b984c00-849c-11ea-8b7e-b5cab9023077.png)

When you are in the tablet version, the articles on the side are overflowing the ones on the left. Please fix this:
![Screenshot from 2020-04-22 13-27-52](https://user-images.githubusercontent.com/51926937/80019451-2a24ad00-849d-11ea-8523-370973ae460e.png)

On the mobile version this section doesn't look like the original page:
The top two articles should have 100% width.
Yours:
![Screenshot from 2020-04-22 13-37-23](https://user-images.githubusercontent.com/51926937/80020380-8d630f00-849e-11ea-9321-60cfe741bfe1.png)

Original:
![Screenshot from 2020-04-22 13-37-13](https://user-images.githubusercontent.com/51926937/80020393-918f2c80-849e-11ea-9f53-2d5583beafc6.png)